src/main/resources/migration/release-0.1.0.xml

```diff
@@ -5,5 +5,15 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
 		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
 
+    <changeSet author="Patokov_N" id="create_basic_tables">
+        <sqlFile dbms="postgresql" path="sql/001_Basic_tables.sql" relativeToChangelogFile="true" />
+        <rollback>
+            DROP TABLE messaging.chat_room_members;
+            DROP TABLE messaging.chat_room_messaging;
+            DROP TABLE messaging.chat_room_info;
+            DROP TABLE messaging.dialog;
+            DROP TABLE messaging.person;
+        </rollback>
```

Comment:
If we start using SQL scripts separately from our XML changelog, it would be right to keep it consistent and move rollback SQL scripts separately too